core/tests/protractor_utils/AdminPage.js

```diff
@@ -258,25 +271,21 @@ var AdminPage = function() {
   };
 
   this.getUsersAsssignedToRole = async function(role) {
-    await waitFor.visibilityOf(
-      roleDropdown, 'View role dropdown taking too long to be visible');
-    await roleDropdown.sendKeys('By Role');
+    await action.sendKeys('Role Drop Down', roleDropdown, 'By Role');
 
-    await roleValueOption.click();
-    await roleValueOption.sendKeys(role);
+    await action.click('Role Value Option', roleValueOption);
```

Comment:
This is redundant because `action.sendKeys(..)` does this internally. See

https://github.com/oppia/oppia/blob/fc48b2c1cd295a8790247aad3ec52896f5741db0/core/tests/protractor_utils/action.js#L52-L55